apps/rellotge/rellotge.js

```diff
@@ -0,0 +1,115 @@
+// Example application code
+// Taken from https://github.com/espruino/BangleApps/blob/master/apps/sclock/clock-simple.js
+(function() {
+
+  const timeFontSize = 1;
+  const dateFontSize = 2;
+  const gmtFontSize = 10;
+  const font = "12x20";
+
+  const xyCenter = g.getWidth() /9;
+  const yposTime = 55;
+  const yposDate = 130;
+  const yposYear = 175;
+  const yposGMT = 220;
+  const leshores = ["Les dotze","La una","Les dues","les tres","Les quatre","Les cinc","Les sis","Les set","Les vuit","Les nou","Les deu","Les onze","Les dotze","La una","Les dues","Les tres","Les quatre","Les cinc","Les sis","Les set","Les vuit","Les nou","Les deu","Les onze","Les dotze"];
```

Comment:
"La una" should be "L'una"
"les tres" should "Les tres" (using upper case like the rest for consistency)